### ISSUE TITLE
when using joycons in controller mode a and b button are switched

### ISSUE THREAD

**Issue by joeyjo33** [Davidobot/BetterJoy#930]
**Describe the bug**
when in the steam big button mode the a and b butttons somehow are switched up. like the a and b buttons on a xbox controller

**Expected behavior**
that betterjoy gets a function in witch you can swap diffrent buttons around to fix this kind of issues.

**Controller (please complete the following information):**
 - Type: joycons in controller mode
 - Connection: bt

**Comment by joeyjo33** [comment]
im gonna try it out. will this also work for the x and y button? same thing
but diffrent line?

Op wo 9 feb. 2022 17:17 schreef DewOnTheGrass ***@***.***>:

**Comment by DewOnTheGrass** [comment]
Yep! There should be another line below it for swapping X and Y.
![swap](https://user-images.githubusercontent.com/36544994/153438429-849d13b5-dc4c-44c4-8a2b-a29cd8e87fd4.PNG)

**Comment by joeyjo33** [comment]
thx im testing it now

Op do 10 feb. 2022 om 16:21 schreef DewOnTheGrass ***@***.***

**Comment by eqagunn** [comment]
Wow. I wish this would have been in the GUI. I was unaware that config file contained additional options and spent quite some time trying to find another tool that would allow me to swap A-B and X-Y. Microsoft's Xbox Accessories app was supposed to be the one to do it, but it doesn't support Xbox360 controller, and BetterJoy emulates it as such. Well, thank you DewOnTheGrass.

**Comment by SgtSweetieBelle** [comment]
Wish it was a GUI setting and also that it could be adjusted on a per controller basis
